webpack.common.config.js

```diff
@@ -25,6 +25,7 @@ var entry = {
     // JS
     'base-page': staticPath('js/pages/base-page.js'),
     'home-page': staticPath('js/pages/home-page.js'),
+    'landing-page': staticPath('js/pages/landing-page.js'),
```

Comment:
`/` is the landing_page, which I think is what you see before you've logged in?   

`/home/` is going to be a page with lots of widgets, including my quick search projects which you are reviewing here, also new and noteworthy projects which you've made some comments on, and an activity widget

`/dashboard/` is Caner's project organizer.

This is what's planned for the home page:
![stage 1](https://cloud.githubusercontent.com/assets/9755598/12618710/16666b9c-c4e3-11e5-8c66-c6eb72be245d.png)